wandb/sdk/internal/internal_api.py

```diff
@@ -169,6 +171,46 @@ class _OrgNames(NamedTuple):
     display_name: str
 
 
+@dataclass
+class Feature:
+    name: str
+    is_enabled: bool
+
+
+@dataclass
+class ServerFeatures:
```

Comment:
i see that it wasn't documented in the original pr boo... mind adding a short docstring?